tests/test_html.snapshot.html

```diff
@@ -385,13 +385,13 @@
 </span></div><div ><span style="font-size: 16.6pt" class="underline bold">Target Value</span><span style="font-size: 16.2pt" class="underline"> 
 </span></div><div ><span style="font-size: 10.7pt" class="italic">Total GFA of Energy Metered 
 </span></div><div ><span style="font-size: 10.7pt" class="underline italic">Data Centers</span><span style="font-size: 10.8pt" class="underline"> 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Total GFA of All Tiered Data 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Centers 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 0.6pt" class="italic"> 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="underline italic">Total Energy Used</span><span style="font-size: 10.8pt" class="underline"> 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Total IT Equipment Energy 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Used 
-</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 0.6pt"> 
```

Comment:
This just reverts all the changes to the snapshot that are currently in #36, so if this gets merged into #36, the net effect should show that the snapshots are unchanged relative to `master`.